docker/docker-compose.yml

```diff
@@ -97,10 +97,17 @@ services:
       DB_NAME: postgres
       DB_USER: postgres
```

Comment:
Should this one be `supabase_realtime_admin` now, or something similar?

Reply:
@kiwicopple we don't have a realtime admin role. `postgres` user works here as long as the db container runs as `postgres` user.